### PR TITLE
Refactor/disable multiple variant callers

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Changed:
- Disable multiple variant callers via comma separation

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: BALSAMIC/commands/run/analysis.py [hunk 1]
```diff
@@ -93,9 +93,12 @@
     help='cluster mail type to send out email. \
               This will be applied to all jobs and override snakemake settings.'
 )
-@click.option('--disable-variant-caller',
-              type=click.Choice(list(VCF_DICT.keys())),
-              help='Run workflow with selected variant caller disable.')
+@click.option(
+    '--disable-variant-caller',
+    help=
```
Comment (ashwini06) [inline BALSAMIC/commands/run/analysis.py:98]: dont you want to have `type` parameter here?

File: BALSAMIC/commands/run/analysis.py [hunk 2]
```diff
@@ -93,9 +93,12 @@
...
+    f'Run workflow with selected variant caller(s) disable. Use comma to remove multiple variant callers. Valid '
+    f'valules are: {list(VCF_DICT.keys())}',
```
Comment (ashwini06) [inline BALSAMIC/commands/run/analysis.py:100]:
```suggestion
    f'values are: {list(VCF_DICT.keys())}',
```

**Review by ashwini06 — COMMENTED** [review]
just one general comment

File: BALSAMIC/commands/run/analysis.py (hunk 1 above)
Comment (hassanfa) [reply BALSAMIC/commands/run/analysis.py:98]: Good point! Default is string but it's good to add it.

Comment (hassanfa) [reply BALSAMIC/commands/run/analysis.py:98]: Other Click options for Click.str() are not set, for sake of consistency I keep it the same here as well. Hope it's ok.

**Pull request merged**